code/game/objects/items/defib.dm

```diff
@@ -239,15 +239,15 @@
 	return ..()
 
 /obj/item/defibrillator/proc/deductcharge(chrgdeductamt)
-	if(cell)
-		if(cell.charge < (paddles.revivecost+chrgdeductamt))
-			powered = FALSE
-			update_power()
-		if(cell.use(chrgdeductamt))
-			update_power()
-			return TRUE
-		else
-			return FALSE
+	if(QDELETED(cell))
+		return
+
+	if(cell.charge < (paddles.revivecost + chrgdeductamt))
+		powered = FALSE
+	else if(!cell.use(chrgdeductamt))
+		return
```

Comment:
```suggestion
	if(cell.charge < (paddles.revivecost + chrgdeductamt))
		powered = FALSE
	if(!cell.use(chrgdeductamt))
		powered = false
```

I think the first check is to set powered to false when there isn't going to be enough energy to do another one. We shouldn't do an else if that is the case.